src/components/dls/Forms/RadioGroup/RadioGroup.tsx

```diff
@@ -32,9 +32,13 @@ interface Props {
   required?: boolean;
   loop?: boolean;
   orientation?: RadioGroupOrientation;
+  renderItem?: (item: RadioItem) => React.ReactNode;
 }
 
-const RadioGroup: React.FC<Props> = ({
+const RadioGroup: React.FC<Props> & {
+  Item: (props: RadioGroupPrimitive.RadioGroupItemProps) => React.ReactElement;
+  Indicator: (props: RadioGroupPrimitive.RadioIndicatorProps) => React.ReactElement;
```

Comment:
Salam bro @ahmedriad1 , thanks making this PR

```
Item: (props: RadioGroupPrimitive.RadioGroupItemProps) => React.ReactElement;
Indicator: (props: RadioGroupPrimitive.RadioIndicatorProps) => React.ReactElement;
```

is this part required. In Modal.tsx, we don't have typing. Maybe we should follow the same approach? what do you think?